sunpy/coordinates/ephemeris.py

```diff
@@ -326,7 +326,8 @@ def get_horizons_coord(body, time='now', id_type=None, *, include_velocity=False
     log.debug(f"See the raw output from the JPL HORIZONS query at {query.uri}")
 
     if isinstance(time, dict):
-        obstime = parse_time(result['datetime_jd'], format='jd', scale='tdb')
+        obstime_tdb = parse_time(result['datetime_jd'], format='jd', scale='tdb')
+        obstime = obstime_tdb.utc
```

Comment:
```suggestion
        obstime = Time(obstime_tdb, format='isot', scale='utc')
```
Also add `from astropy.time import Time` up top.  I'd wager that most solar users would rather see times in ISOT format than in JD format.